doc/README.md

```diff
@@ -0,0 +1,21 @@
+# pyglmnet documentation
+
+See full documentation page [here](http://pavanramkumar.github.io/pyglmnet/).
```

Comment:
I am wondering if this should go in the description of the github page. Basically where we say: "Python implementation of elastic-net regularized generalized linear models":

![screen_pyglmnet](https://cloud.githubusercontent.com/assets/15852194/15273095/37bf6328-1a8f-11e6-8b40-f992cbb3b285.png)